sys/userlib/src/kipc.rs

```diff
@@ -162,3 +162,41 @@ pub fn software_irq(task: usize, mask: u32) {
         &[],
     );
 }
+
+/// Reads a task's panic message into the provided `buf`, if the task is
+/// panicked.
+///
+/// Note that Hubris only preserves the first [`PANIC_MESSAGE_MAX_LEN`] bytes of
+/// a task's panic message, and panic messages greater than that length are
+/// truncated. Thus, this function accepts a buffer of that length.
```

Comment:
Just wanted to note that we could most likely loosen this later, if required, by changing this to a `&mut [u8]` and not breaking callers.

Reply:
Yup, that was my thinking as well.